tests/ui_tests/app/page/job/page.py

```diff
@@ -76,3 +88,44 @@ class JobPageStderr(JobPageMixin):
     """Job Page Stderr log"""
 
     MENU_SUFFIX = '2'
+
+
+class JobPage(BasePageObject):
```

Comment:
Pls, have a look at JobPageMixin and JobPageStdout and JobPageStderr

Reply:
Thanks, reworked to Stdout version